ompi/win/win.c

```diff
@@ -124,7 +124,6 @@ int ompi_win_init (void)
 
     /* Setup MPI_WIN_NULL */
     OBJ_CONSTRUCT(&ompi_mpi_win_null.win, ompi_win_t);
-    ompi_mpi_win_null.win.w_flags = OMPI_WIN_INVALID;
```

Comment:
Is this safe to do?  Are there other places that rely on this flag upon construction?

Reply:
That's exactly what I said above (and it is true for all uses of _INVALID). But I don't think it should be part of this PR.